pulp_smash/pulp_smash_cli.py

```diff
@@ -196,5 +197,57 @@ def settings_validate(ctx):
         raise result
 
 
+@pulp_smash.command('smoke-tests')
+def smoke_tests():
+    """Print smoke tests, one per line.
+
+    Sample usage:
+
+        python -m unittest $(pulp-smash smoke-tests)
+    """
+    test_suite = unittest.TestSuite()
+    test_suite.addTests(unittest.TestLoader().discover('pulp_smash.tests'))
```

Comment:
@Ichimonji10, Can we restricted the discover just for Pulp3 namespace?

Reply:
What's the benefit in doing so?

Reply:
It is just matter to avoid to parse the whole Pulp2 namespace looking for smoke tests, since right now we do not have any.  Besides that, I do not see any other.

Reply:
I'd like to search the entire Pulp namespace by default. If efficency becomes a concern, then we can expand the user interface, so that the user can execute commands like:

```bash
pulp-smash smoke-tests pulp_smash.tests.pulp2
pulp-smash smoke-tests pulp_smash.tests.pulp3
pulp-smash smoke-tests pulp_smash.tests.pulp3.pulpcore
```

I do expect that we will mark some Pulp 2 tests as being smoke tests. When that happens, it'd be nice if they just showed up in the output of `pulp-smash smoke-tests` without us having to fiddle with the CLI machinery.

Reply:
@Ichimonji10, thanks.